docs/tutorials/Canopy/canopy_tutorial.jl

```diff
@@ -114,11 +114,11 @@ shared_params = SharedCanopyParameters{FT, typeof(earth_param_set)}(
 
 ψ_soil0 = FT(0.0)
 
-soil_driver = PrescribedSoil(
-    root_depths = FT.(-Array(10:-1:1.0) ./ 10.0 * 2.0 .+ 0.2 / 2.0),
-    ψ_soil = t -> eltype(t)(ψ_soil0),
-    soil_α_PAR = FT(0.2),
-    soil_α_NIR = FT(0.4),
+soil_driver = PrescribedSoil{FT}(
+    FT.(-Array(10:-1:1.0) ./ 10.0 * 2.0 .+ 0.2 / 2.0), # root_depths
```

Comment:
out of curiosity, do we need these FTs in the arguments?

Reply:
I wanted to enforce that they'll all be FT, but I think it's sufficient to pass FT as a type argument to `PrescribedSoil`, so I can remove it for the individual arguments

Reply:
btw, there are some places in main where we do this currently (like [here](https://github.com/CliMA/ClimaLSM.jl/blob/main/docs/tutorials/Canopy/canopy_tutorial.jl#L127-L137)), but I think it's okay to have a mix of approaches and leave it out here

Reply:
ok! and good to keep in mind. as I come across those, Ill remove them as long as they dont clutter PR diffs!